custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -57,7 +57,7 @@
         <script src="{% static 'js/topojsons/districts_v2.topojson.js' %}"></script>
         <script src="{% static 'js/topojsons/blocks_v3.topojson.js' %}"></script>
     {% else %}
-        <script src="{% static 'js/topojsons/states.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/state.topojson.js' %}"></script>
```

Comment:
What does this PR do? It seems to change two files that are behind a feature flag, then change a file reference unrelated to those?

Reply:
@calellowitz this change should be reverted. I did this today morning.